app/src/main/java/dev/hossain/remotenotify/ui/alertmediumlist/WorkerCheckIntervalConfigUi.kt

```diff
@@ -36,7 +36,11 @@ internal fun WorkerConfigCard(
     state: NotificationMediumListScreen.State,
     modifier: Modifier = Modifier,
 ) {
-    var intervalSliderValue by remember { mutableFloatStateOf(state.workerIntervalMinutes.toFloat()) }
+    // Update intervalSliderValue whenever state.workerIntervalMinutes changes
+    var intervalSliderValue by remember(state.workerIntervalMinutes) {
```

Comment:
The `remember(state.workerIntervalMinutes)` was the fix. Lots of AI iteration with no solution. Rare failure.